src/QsCompiler/Transformations/CallGraph.cs

```diff
@@ -2,11 +2,9 @@
 // Licensed under the MIT License.
```

Comment:
Look into using the #nullable enable feature to clean up null checks across this feature. @samarsha for the idea.

Reply:
I've added the `#nullable enable` feature to the call graph file, although it wasn't able to reduce any of the null checks in the file because public facing members still need to have the null checks.